pages/members/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { observer } from 'mobx-react';
+import { PropsWithoutRef, PureComponent } from 'react';
+import { Container } from 'react-bootstrap';
+import { Loading } from 'idea-react';
+
+import PageHead from '../../components/PageHead';
+import membersStore, { Member } from '../../models/Members';
+import { LazyLoad } from '../../utils/layzload';
+import MembersTitle from '../../components/Members/MembersTitle';
+import MembersList from '../../components/Members/MembersList';
+import MembersTabs, {
+  MembersTabsProps,
+} from '../../components/Members/MembersTabs';
+
+type MembersGroup = PropsWithoutRef<{
+  [proppName: string]: MembersTabsProps;
+}>;
+
+@observer
+export default class MembersPage extends PureComponent {
+  state: Readonly<{ membersGroup: MembersGroup; otherMembersList: Member[] }> =
+    {
+      membersGroup: {},
+      otherMembersList: [],
+    };
```

Comment:
```suggestion
  @observable
  membersGroup = {} as MembersGroup;

  @observable
  otherMembersList: Member[] = [];
```
直接用 MobX 更简洁：https://github.com/mobxjs/mobx/blob/mobx4and5/docs/refguide/observer-component.md#local-observable-state-in-class-based-components